cdci_data_analysis/analysis/parameters.py

```diff
@@ -674,29 +679,27 @@ def _set_angle(self, value, units):
 #
 
 
+class SpectralBoundary(Float):
+    def __init__(self, value=None, E_units='keV', name=None):
 
-class SpectralBoundary(Parameter):
-    def __init__(self,value=None,E_units='keV',name=None):
-
-        _allowed_units = ['keV','eV','MeV','GeV','TeV','Hz','MHz','GHz']
-
+        _allowed_units = ['keV', 'eV', 'MeV', 'GeV', 'TeV', 'Hz', 'MHz', 'GHz']
```

Comment:
this should be set in `Energy`

Reply:
so, extend the `_allowed_units` of the `Energy class?

Reply:
yes please!